docs/user-guide.md

```diff
@@ -0,0 +1,223 @@
+# OpenShift Appliance User Guide
+
+## What is OpenShift Appliance
+The `openshift-appliance` is a container-based utility that builds a disk image that facilitates the installation of OpenShift with an [Agent-based installer](https://cloud.redhat.com/blog/meet-the-new-agent-based-openshift-installer-1).
+
+Its main purpose is to enable a completely offline installation of an OpenShift cluster, Thus, all required images are included in the appliance disk image.
+
+
+## Download
+OpenShift Appliance is available for download at: https://quay.io/edge-infrastructure/openshift-appliance
+
+## High-Level Flow Overview
+
+![hl-overview.png](images%2Fhl-overview.png)
+
+### Lab
+* This is where `openshift-appliance` gets used to create a raw space disk image.
+  * **raw:** so it can be copied as-is to multiple servers. 
+  * **sparse:** to keep the size as small as possible.
+* The end result is a generic disk image with a partition layout as follows:
+  ``` bash
+  Name       Type        VFS      Label       Size  Parent
+  /dev/sda2  filesystem  vfat     EFI-SYSTEM  127M  -
+  /dev/sda3  filesystem  ext4     boot        350M  -
+  /dev/sda4  filesystem  xfs      root        180G  -
+  /dev/sda5  filesystem  ext4     agentboot   1.2G  -
+  /dev/sda6  filesystem  iso9660  agentdata   18G   -
```

Comment:
Is this still enough? 
I wonder if https://issues.redhat.com/browse/MGMT-15165 is because of that @danielerez

Reply:
@achuzhoy which partition specifically?
@danielerez, can you comment on that?

Reply:
The agentdata partition is read-only, it's size is calculated according the release images. I.e. its size statically defined on build.